db/seeds.rb

```diff
@@ -5,3 +5,37 @@
+zoe = User.create!(email: "zoe@example.com", password: "123456", nickname: "zoe-bowie", bio: "I work in the IT industry for 20 years now. I love share my experience with you and to help you with your upcoming interviews!!")
+peter = User.create!(email: "peter@example.com", password: "123456", nickname: "coding-king", bio: "I am the KING OF CODING and I will show you how it works so you get every job you want!")
+lisa = User.create!(email: "lisa@example.com", password: "123456", nickname: "lizzy101", bio: "studied in England, worked as developer in Seoul, worked as senior develper in Tokio")
+bruce = User.create!(email: "bruce@example.com", password: "123456", nickname: "brucybruce", bio: "I got 10 years of experience in Ruby and Ruby on rails, CSS, JS and Python. I can help you prepare for your job interview!")
+puts "generating Interviews"
+interview1 = Interview.create!(time: DateTime.new(2009,9,1,17), focus: "Job for WebDev", experience: "medium", interview_language: "english", user: john)
+interview2 = Interview.create!(time: DateTime.new(2020,12,12,17), focus: "Backend WebDev", experience: "entry level", interview_language: "german", user: mary)
+interview3 = Interview.create!(time: DateTime.new(2020,12,12,18), focus: "General Coding Interview", experience: "Senior level", interview_language: "french", user: zoe)
+interview4 = Interview.create!(time: DateTime.new(2009,9,1,17), focus: "Google Cloud Interview", experience: "medium", interview_language: "japanese", user: peter)
+interview5 = Interview.create!(time: DateTime.new(2020,12,12,17), focus: "Developer Database", experience: "beginner", interview_language: "english", user: lisa)
+interview6 = Interview.create!(time: DateTime.new(2020,12,12,18), focus: "IT Consultant", experience: "Senior level", interview_language: "dutch", user: bruce)
+interview7 = Interview.create!(time: DateTime.new(2009,9,1,17), focus: "Ruby on Rails developer", experience: "expert", interview_language: "french", user: john)
+interview8 = Interview.create!(time: DateTime.new(2020,12,12,17), focus: "Head of Development", experience: "medium", interview_language: "english", user: mary)
+interview9 = Interview.create!(time: DateTime.new(2020,12,12,18), focus: "Frontend WebDev", experience: "Senior level", interview_language: "german", user: zoe)
+interview10 = Interview.create!(time: DateTime.new(2009,9,1,17), focus: "PHP developer", experience: "beginner", interview_language: "english", user: peter)
+interview11 = Interview.create!(time: DateTime.new(2020,12,12,17), focus: "PHP", experience: "medium", interview_language: "english", user: lisa)
+interview12 = Interview.create!(time: DateTime.new(2020,12,12,18), focus: "General IT Job", experience: "Senior level", interview_language: "spanish", user: bruce)
```

Comment:
The focus attribute will contain our tags in the end, i.e. Ruby, CSS, JS, HTML. For the demo I suggest we implement this in the seed by just assigning 1 of these 4 as focus.

Reply:
easy change. will be done!